runtime/near-vm-runner/src/wasmer2_runner.rs

```diff
@@ -286,53 +274,54 @@ fn run_method(module: &Module, import: &ImportObject, method_name: &str) -> Resu
     Ok(())
 }
 
-pub(crate) fn compile_wasmer1_module(code: &[u8]) -> bool {
-    let store = default_wasmer1_store();
+pub(crate) fn compile_wasmer2_module(code: &[u8]) -> bool {
+    let store = default_wasmer2_store();
     Module::new(&store, code).is_ok()
 }
 
 #[derive(Hash)]
-struct Wasmer1Config {
+struct Wasmer2Config {
     seed: i32,
     use_cranelift: bool,
     use_native_engine: bool,
 }
 
-impl Wasmer1Config {
+impl Wasmer2Config {
     fn config_hash(self: Self) -> u64 {
         let mut s = DefaultHasher::new();
```

Comment:
Unrelated to this PR, but I now realize that using `DefaultHasher` is suboptimal here -- Rust doesn't guarantee stability of the hasher, so, rust version upgrade might bust the cache. 

We should steal this module from Cargo: 

https://github.com/rust-lang/cargo/blob/58a961314437258065e23cb6316dfc121d96fb71/src/cargo/util/hasher.rs

This will both guarantee hash stability in the future, as well as compatibility with the current hash we de-facto use.

cc @olonho